stories/toggle/toggle-accessibility.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Toggle } from "src/toggle";
+import { Typography } from "src/typography";
+import styled from "styled-components";
+import { Colour } from "src/theme";
+
+type Component = typeof Toggle;
+
+const meta: Meta<Component> = {
+    title: "Selection and input/Toggle/Accessibility Guidelines",
+    component: Toggle,
+};
+export default meta;
+
+const ErrorText = styled(Typography.BodyMD)`
+    color: ${Colour["text-error"]};
+`;
+
+export const GroupExample: StoryObj<Component> = {
```

Comment:
currently navigating the radio group with arrow keys ends up selecting both items

setting `defaultChecked` also doesn't seem to do anything? perhaps this needs a `useState` to track the current selection?